oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -431,6 +431,18 @@ promql:
     restMaxThreads: ${SW_PROMQL_REST_MAX_THREADS:200}
     restIdleTimeOut: ${SW_PROMQL_REST_IDLE_TIMEOUT:30000}
     restAcceptQueueSize: ${SW_PROMQL_REST_QUEUE_SIZE:0}
+    
+#This module is for LogQL API.
+logql:
+  selector: ${SW_LOGQL:default}
+  default:
+    # For HTTP server
+    restHost: ${SW_LOGQL_REST_HOST:0.0.0.0}
+    restPort: ${SW_LOGQL_REST_PORT:3100}
```

Comment:
With this, `showcase` and `helm` codes need to be updated to expose the new port, right? @kezhenxu94 @wankai123

Reply:
I suggest we don't enable logql, and promql which I missed before, they are occupying too many ports and are not common cases where users would use, just let the users enable these when they want

Reply:
I am fine about enabling or not by default. It seems not much different. Are you concern about network security impact?

Reply:
> I am fine about enabling or not by default. It seems not much different. Are you concern about network security impact?

it's much different, enabling an unused plugin creates an extra HTTP server, consuming more resources, and security impact is the most important concern of mine.